### PR TITLE
[core] adjust oom defaults

### PR DESCRIPTION
**Pull request by clarng**
Signed-off-by: Clarence Ng <clarence.wyng@gmail.com>

## Why are these changes needed?

Per discussion, only use fractional threshold, with the assumption more memory -> more cpu

## Related issue number

https://github.com/ray-project/ray/issues/30433

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [x] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by clarng** [comment]
Nightly tests: https://buildkite.com/ray-project/release-tests-pr/builds?branch=clarng%3Aoomdef

File: doc/source/ray-core/scheduling/ray-oom-prevention.rst [hunk 1]
```diff
@@ -25,13 +25,13 @@ How do I configure the memory monitor?
...
-- ``RAY_memory_monitor_interval_ms (int, defaults to 9)`` is the interval to check memory usage and kill tasks or actors if needed. It is disabled when this value is 0.
+- ``RAY_memory_monitor_interval_ms (int, defaults to 250)`` is the interval to check memory usage and kill tasks or actors if needed. It is disabled when this value is 0.
 
-- ``RAY_memory_usage_threshold_fraction (float, defaults to 0.9)`` is the threshold when the node is beyond the memory
+- ``RAY_memory_usage_threshold_fraction (float, defaults to 0.95)`` is the threshold when the node is beyond the memory
   capacity. If the memory usage is above this value and the free space is
   below min_memory_free_bytes then it will start killing processes to free up space. Ranges from [0, 1].
 
-- ``RAY_min_memory_free_bytes (int, defaults to 1 GiB)`` is the minimum amount of free space. If the memory usage is above
+- ``RAY_min_memory_free_bytes (int, defaults to -1)`` is the minimum amount of free space. If the memory usage is above
```
Comment (ericl) [inline doc/source/ray-core/scheduling/ray-oom-prevention.rst:34]: Could we hide this config from the public docs?

Comment (clarng) [reply doc/source/ray-core/scheduling/ray-oom-prevention.rst]: done

**Comment by clarng** [comment]
Moved to https://github.com/ray-project/ray/pull/30553

**Pull request closed without merging**